### PR TITLE
Update TimeIntervalPicker.swift

### PR DESCRIPTION
**Pull request by safakge**
Fixed a bug where the picker would set the hour component instead of the minute when selectedMinute < 60

### CODE REVIEW

**Comment by ChernyshenkoTaras** [comment]
Hi @safakge I'm really appreciate your help. But did you test your solution? Your code rise exception 
`Terminating app due to uncaught exception 'NSRangeException', reason: '*** -[__NSArrayM objectAtIndex:]: index 1 beyond bounds [0 .. 0]'`

Nevertheless, could you provide more info about bug? Right now, when you set minutes less then 60, `dialog` would show only minutes component.

Hours component will appear only if `maxMinutes` will be greater then 60 minutes.

**Comment by safakge** [comment]
Hi @ChernyshenkoTaras, thank you for the library. You are right, I only tested for my own case, where `maxMinutes = 60*24` and picker is initialized with `show(at:5)`. The original code sets the picker incorrectly at 5:00 where it should've been 0:05. My modification fixed this, but also broke it for the case where `maxMinutes < 60`. Didn't realize the picker showed a single component in this case.

In short, both of our codes are incorrect. :) Please ignore this for now, I will send another patch.
